### ISSUE TITLE
Still an auto-search issue with 1.10.2

### ISSUE THREAD

**Issue by sveakul** [jacquesh/foo_openlyrics#389]
Thanks for your fast notification, but unfortunately there is still a problem in 1.10.2--when the panel is closed, no autosearch happens (hurrah!) if that option is not selected, but when it is tabbed open, no autosearch begins until "Search for Lyrics" is selected from the context menu manually.

It does begin an autosearch if it is the only tab open at the time.  But if it is closed while a stream is playing, then opened, it does not instigate an auto-search on its own.

Better like this than before (thank you!) and definitely can be lived with--but there must be one small part of the fix that got overlooked.

**Comment by jacquesh** [comment]
Correct, this is known (in some sense expected) behaviour. As far as I'm aware this is the same as the behaviour you get if you add an openlyrics panel to the UI while a track is playing (in v1.10 as well as v1.19) so I'm not going to stress about getting a hotfix out for it. I've added to the UX-focus milestone instead so we'll get to it.

**Comment by jacquesh** [comment]
Haha, no indeed you're not crazy. I know that happens and I know why and I know what I would need to do to change it. When I made this change my reasoning to myself for why it wasn't an issue is that people don't change layouts very often, but I hadn't considered the case of tabbed containers.

I'll get to it, I'm just not considering it super urgent so it'll be in a future release (which realistically means you should not expect a resolution for at least the next several months).

**Comment by jacquesh** [comment]
> I re-installed your component, just a few days ago, and I still experience the same behaviour. Which is perfectly fine, by the way. I'd prefer an automatic search but it works fine as it is.

Yes, I did notice while working on v1.10 that this was still an issue, I just didn't worry about it too much because I expect people don't edit their layout all that often. In the interest of not getting lost in comments, please create a separate issue for resolving this. It should be fairly simple to do, but the cause is different to what sveakul is highlighting above.

> Just one thing to note: If I start a radio stream (waiting a few seconds for the song title and artist to be displayed), proceed to open the Panel (no automatic search performed), then right-click and hit "Search for lyrics" I get no lyrics and see a bunch of errors in console.

As above, please open a separate issue. This is an artifact of the somewhat annoying way that fb2k exposes metadata for internet radio (you have to be there waiting to receive it when it gets downloaded, you can't query for it later). It can be worked around though.

**Comment by snowbie** [comment]
> no autosearch happens (hurrah!) if that option is not selected

Thanks for this. I must've glazed over that setting. The lyrics not loading in a hidden panel bothered me enough that I considered reverting to 1.6 again, since the versions between failed to load local lyrics for me.
